docs/getting-started.md

```diff
@@ -7,22 +7,29 @@ description: Jump in and get started quickly using Momento Serverless Cache with
 ---
 
 # Getting Started with Momento Serverless Cache
-If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. There are multiple ways to access the service below. 
+If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. Want to quickly get a feel for the Momento experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser). Or, if you're ready to setup your development environment and get to building your own applications with Momento you can follow the [simple instructions below](#install-the-momento-command-line-interface-cli).
 
-## Try our CLI in your browser
+## Try our CLI and an SDK in your browser
```

Comment:
I don't think the word "an" is necessary in the header. To me it seems to read better if it's just
```markdown
## Try our CLI and SDK in your browser
```

Reply:
I guess I could go either way on this. But we have multiple SDKs and this demo shows one in use, right?